### PR TITLE
Setup/styling libs

### PR DESCRIPTION
**Pull request by dBanasiak**

### CODE REVIEW

File: website/src/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (KonradLinkowski) [inline website/src/.eslintrc.json]: I'm not sure if eslint works well with typescript. I would consider changing it to tslint

**Review by KonradLinkowski — DISMISSED** [review]
Looks cool

File: website/src/.eslintrc.json (hunk 1 above)
Comment (dBanasiak) [reply website/src/.eslintrc.json]: Ok, sure. Changed to tslint

**Review by KonradBaranek — APPROVED** [review]
Good work!

**Pull request merged**